.travis.yml

```diff
@@ -75,6 +75,16 @@ script:
       presto-product-tests/bin/run_on_docker.sh \
         multinode -x quarantine,big_query,storage_formats,profile_specific_tests,tpcds,cassandra,mysql_connector,postgresql_connector,mysql
     fi
+  - |
```

Comment:
Do not add those tests to a regular build. I think we run them on a different infrastructure. (@findepi do we BTW? =) )

Build fails because there are too much logs for a single build. So either remove them from here, or run them in a separate travis job.

Reply:
@kokosing knows everything

Reply:
So what's the suggestion in general? Maybe break into two jobs?

Reply:
Yes, break into two jobs. Try to make them even. You can move all the impersonation related tests to `PRODUCT_TESTS_SPECIFIC_ENVIRONMENT_2`